tox.ini

```diff
@@ -1,5 +1,31 @@
 [tox]
-envlist = copying,py27{,-color}
+envlist = copying,{py34,py35,py36,py37}-{lint,tests,color,cli}
+
+[testenv]
+whitelist_externals =
+    cli:   {[testenv:cli]whitelist_externals
+deps=
+    git+git://github.com/iot-lab/cli-tools.git#egg=iotlabcli
+    -rtests_utils/test-requirements.txt
+commands=
+    tests:      {[testenv:tests]commands}
+    lint:       {[testenv:lint]commands}
+    cli:        {[testenv:cli]commands}
+    color:      {[testenv:color]commands}
+    coverage:   {[testenv:coverage]commands}
+
+[testenv:tests]
+commands=
+    py.test --cov=iotlabaggregator --cov-report=term --cov-report=xml --cov-report=html
+
+[testenv:color]
+commands=
+    pip install -e .[color_serial]
```

Comment:
is `-e` option really required ? (it creates symlinks)

Reply:
> * change the version number with a breaking change:
>   **version** = '1.5.0'   => 2.xx
>   install_requires=['iotlabcli>=2.1.0'] => requires iotlabcli version 3

@rienafairefr For me we don't need a iotlabcli version 3. We just need a version of iotlabcli which supports python version 3. So 2.2.1 version will be a good option ?

Reply:
you're right, iotlabcli does not need to be version 3 :-)

And the version of aggregation-tools that supports only python 3 should be a breaking version change